setup.py

```diff
@@ -159,6 +159,7 @@
 #
 # some of the values are versioned whereas others aren't.
 deps = {b: a for a, b in (re.findall(r"^(([^!=<>]+)(?:[!=<>].*)?$)", x)[0] for x in _deps)}
+deps["detectron2"] = "detectron2@git+https://github.com/facebookresearch/detectron2.git"
```

Comment:
this isn't right - the string ``detectron2 @ git+https://github.com/facebookresearch/detectron2.git` should be above right below "uvicorn"